ci-operator/step-registry/cucushift/upgrade/toversion/cucushift-upgrade-toversion-commands.sh

```diff
@@ -430,17 +376,8 @@ if (( retry == 0 )); then
 fi
 
 set_target_and_upgrade_cmd
-export FORCE_UPDATE="false"
```

Comment:
Dropping `run_command_oc` sounds good to me. I'd suggest we keep the code on checking signature, we had a [short discussion](https://github.com/openshift/release/pull/49743#discussion_r1525745330) around it before. Actually in the ci watch, we experienced the payload is not signed yet, so `--force` is still needed even for a stable build, this would be more flexible to help the test move on.

Reply:
But sounds like we never reference `FORCE_UPDATE` in `upgrade_cmd`, right?

Reply:
good catch, i think we should add it.

Reply:
Updated.

Though pj-rehearse jobs failed, it failed because not recommended update, the source version - 4.14.29 sounds like a bit new, does not have upgrade path yet.

But I think the PR change should be low risky, can move forwards to merge.